rgw/v2/tests/s3_swift/reusable.py

```diff
@@ -1002,17 +1002,28 @@ def check_datalog_marker():
 
     # fetch the first occurance of marker
     get_datalog_marker = ""
+    shard_id = -1
     datalog_num_shards = int(datalog_num_shards) - 1
     for i in range(datalog_num_shards):
         if datalog_status[i]["marker"] is "":
             continue
         else:
             get_datalog_marker = datalog_status[i]["marker"]
+            shard_id = i
             break
 
-    if "1_" in get_datalog_marker:
+    # return shard_id and datalog_mark, Ref BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1981860
+    return shard_id, get_datalog_marker
+
+
+def check_datalog_marker():
+    """
+    check the datalog marker
+    """
+    _, marker = get_datalog_marker()
+    if "1_" in marker:
         return "omap"
-    if ":" in get_datalog_marker:
+    if ":" in marker:
         return "fifo"
-    if "" in get_datalog_marker:
-        raise TestExecError("failed to fetch datalog marker")
+    #    if "" in marker:
+    raise TestExecError(f"Unkown failed to check datalog marker \n {marker}")
```

Comment:
Sorry. The wording is incorrect. Can we change this to "raise TestExecError(f"Unknown: No known identifiers found in datalog marker \n {marker}")`

Reply:
@psathyan , changed the wording accordingly